wremnants/CardTool.py

```diff
@@ -362,12 +361,12 @@ def addSystematic(self, name, systAxes=[], systAxesFlow=[], outNames=None, skipE
                 "preOpArgs" : preOpArgs,
                 "action" : action,
                 "actionArgs" : actionArgs,
+                "actionRequiresNomi" : actionRequiresNomi,
```

Comment:
I get that we could need this, but can't the 0th entry in the syst axis just be the nominal? It makes things a lot cleaner if it is

Reply:
In this particular case the histogram I start with is a general histogram with same axes as the nominal (just filled with different event weights), so the nominal layer is not present at all, and adding it in the function is useless since eventually the new decorrelated histogram will be split again into the single syst components.

If instead you meant that the "nominal" is the fully correlated syst, this might be implemented, but the disadvantage is that you would get it even when you don't want to keep the inclusive systematic, so that it should be excluded later.

Reply:
What is the histogram? It's the scaling of the process by X% ?

Reply:
It is a histogram equivalent to the nominal but filled with a weight defined as " (some condition) ? (factor*nominalWeight) : nominalWeight)

In practice, since the condition identifies a subset of events, for some processes, it is equivalent to scaling that fraction of events by x% (currently 100%). It could be achieved after the loop if you saved the process split into the components, I was initially doing it to be able to plot them separately and test them, but for the actually analysis the splitting is not necessary since we don't want to see too many processes.